src/Pages/FeedDetail/CommentBox.js

```diff
@@ -0,0 +1,181 @@
+import styled from 'styled-components';
+import { useState } from 'react';
+
+const CommentBox = ({
+  id,
+  addEditComment,
+  comment,
+  formOpen,
+  edittingCommentBoxId,
+  formClose,
+  isLoggedIn,
+  onRemove,
+}) => {
+  const [newComment, setNewComment] = useState(comment.userComment);
+
+  const handleEdit = e => {
+    setNewComment(e.target.value);
+    addEditComment(e.target.value, id);
+  };
+
+  const handleOnRemove = () => {
+    onRemove(id);
+  };
+
+  // const handleOnRemove = () => {
+  //   ;
+  // };
+
+  console.log(comment);
+  return (
+    <>
+      <li className="commentLi">
+        <CommentContainer>
+          <img src="Images/FeedDetail/profile.png" alt="profile"></img>
+          <div className="commentContents">
+            {edittingCommentBoxId !== comment.id ? (
+              <CommentTop>
+                <span className="commentUserId">{comment.userId}</span>
+
+                <span className="commentText">{comment.userComment}</span>
+              </CommentTop>
+            ) : (
+              <FormEdit onBlur={formClose}>
+                <span className="commentUserId">{comment.userId}</span>
+
+                <input value={newComment} onChange={e => handleEdit(e)}></input>
+                <CommentBtn comment={comment} onClick={formClose}>
+                  <span>확인</span>
+                </CommentBtn>
+              </FormEdit>
+            )}
+            {edittingCommentBoxId !== comment.id ? (
+              <CommentInfo>
+                <span className="commentDate">{comment.Date}</span>
+                <span class="bullit">{'・'}</span>
+                <button>좋아요</button>
+                <span class="bullit">{'・'}</span>
+                <button>답글달기</button>
+                {isLoggedIn && (
+                  <>
+                    <span class="bullit">{'・'}</span>
+                    <button onClick={() => formOpen(comment.id)}>
+                      수정하기
+                    </button>
+                    <span class="bullit">{'・'}</span>
+                    <button onClick={() => handleOnRemove(id)}>삭제하기</button>
+                  </>
+                )}
+              </CommentInfo>
+            ) : null}
```

Comment:
```suggestion
            {edittingCommentBoxId !== comment.id && (
              <CommentInfo>
                <span className="commentDate">{comment.Date}</span>
                <span class="bullit">{'・'}</span>
                <button>좋아요</button>
                <span class="bullit">{'・'}</span>
                <button>답글달기</button>
                {isLoggedIn && (
                  <>
                    <span class="bullit">{'・'}</span>
                    <button onClick={() => formOpen(comment.id)}>
                      수정하기
                    </button>
                    <span class="bullit">{'・'}</span>
                    <button onClick={() => handleOnRemove(id)}>삭제하기</button>
                  </>
                )}
              </CommentInfo>
            )}
```
조건에 따라 나타나는 결과가 한 가지만 있다면 3항 연산자가 아닌 &&를 사용해보세요!